pip-wheel-metadata/hera_sim.dist-info/AUTHORS.rst

```diff
@@ -0,0 +1,5 @@
+=======
```

Comment:
Are you sure you want all this pip-wheel-metadata stuff committed into the repo? I think it's autogenerated...

Reply:
Ah, danmit. Have to take that `git` course again...